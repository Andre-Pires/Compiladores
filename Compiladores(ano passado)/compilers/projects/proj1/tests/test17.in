typedef struct {
  int a;
  int b[10];
  int *ptr;
}

