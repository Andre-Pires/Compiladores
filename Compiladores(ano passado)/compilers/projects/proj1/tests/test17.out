1 270 typedef
1 274 struct
1 123 {
2 271 int
2 257 a
2 59 ;
3 271 int
3 257 b
3 91 [
3 258 10
3 93 ]
3 59 ;
4 271 int
4 42 *
4 257 ptr
4 59 ;
5 125 }
#Total Number of Tokens Matched was 17
